model_analyzer/triton/client/client.py

```diff
@@ -74,7 +74,7 @@ def wait_for_server_ready(
             "Could not determine server readiness. " "Number of retries exceeded."
         )
 
-    def load_model(self, model_name):
+    def load_model(self, model_name, variant_name, config_str=None):
```

Comment:
What is the purpose of `variant_name` here? Do we need this if it is only used for logging? 

I think we can at least make it optional so that we avoid a call that looks like 
```python
load_model("model_name", "model_name")
```

Reply:
Yes. Good point. I've made it optional.